lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -369,8 +369,8 @@ struct Emitter {
   void emitCalyxControl(Block *body) {
     Operation *parent = body->getParentOp();
     assert(isa<ControlOp>(parent) ||
-           parent->hasTrait<ControlLike>() &&
-               "This should only be used to emit Calyx Control structures.");
+           (parent->hasTrait<ControlLike>() &&
+            "This should only be used to emit Calyx Control structures."));
```

Comment:
I think it makes a bit more sense to group the left part of this:
```c++
    assert((isa<ControlOp>(parent) ||
           parent->hasTrait<ControlLike>()) &&
            "This should only be used to emit Calyx Control structures."));
```

Reply:
Thanks for reviewing.